src/ASM/LR/ASMu3D.C

```diff
@@ -1982,16 +1990,41 @@ bool ASMu3D::evalSolution (Matrix& sField, const IntegrandBase& integrand,
     fe.v   = gpar[1][i];
     fe.w   = gpar[2][i];
     fe.iel = iel+1;
+
+    double du = lrspline->getElement(iel)->umax() - lrspline->getElement(iel)->umin();
+    double dv = lrspline->getElement(iel)->vmax() - lrspline->getElement(iel)->vmin();
+    double dw = lrspline->getElement(iel)->wmax() - lrspline->getElement(iel)->wmin();
+
+    double u[2*p1];
+    double v[2*p2];
+    double w[2*p3];
+    basis1.computeBasisValues(gpar[0][i], u, 1);
+    basis2.computeBasisValues(gpar[1][i], v, 1);
+    basis3.computeBasisValues(gpar[2][i], w, 1);
+
+    size_t ib=1; // basis function iterator
+    for(int k=0; k<p3; k++) {
+      for(int j=0; j<p2; j++) {
```

Comment:
I would like these to be `jj` and `kk` just for consistency.

Reply:
done.